packages/yourloops/pages/profile/profile.tsx

```diff
@@ -360,14 +365,15 @@ export const ProfilePage: FunctionComponent = () => {
                 error={errors.birthDate}
                 helperText={errors.birthDate && t("required-field")}
               />
-              <TextField
-                id="hbA1c"
-                label={t("hcp-patient-profile-hba1c")}
-                disabled
-                value={hbA1c}
-                onChange={handleChange(setHbA1c)}
-                className={classes.textField}
-              />
+              {hbA1c && (
```

Comment:
I don't really like this construction.
- It's a comparison with 2 different things (string && object) => && should be reserved for boolean.
- It's easy to read it wrong.

Reply:
I don't really understand your point. The [reactjs's documentation](https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator) recommends it

Reply:
Yes but, as they say:
> It works because in JavaScript, true && expression always evaluates to expression
It's a JavaScript odd behavior.

I just don't like this approach. But if you're happy with it, let's say it's OK

Reply:
What's odd in this behavior ?